### ISSUE TITLE
1.18: startup issue

### ISSUE THREAD

**Issue by broken1arrow** [kangarko/Foundation#125]
The issue is this https://pastebin.com/SjtBEy0H i think  Foundation do something before my plugin load.

I change it to on enable and it start with no issue. I can not rely test if it only issue, because when I change I can´t use commands methods like registerCommand and registerCommands any more (will look on register commands my self and see if other parts works).

**Comment by broken1arrow** [comment]
I think this is pretty imported to fix, so we can test if we need fix own code to. 

And if you can, fix better debug msg so we know what the issue is.

**Comment by broken1arrow** [comment]
> The issue is this https://pastebin.com/SjtBEy0H i think Foundation do something before my plugin load.
> 
> I change it to on enable and it start with no issue. I can not rely test if it only issue, because when I change I can´t use commands methods like registerCommand and registerCommands any more (will look on register commands my self and see if other parts works).

I think I get your issue, i think it can´t load one class from your plugin (I think a version check some stop it to run).

**Comment by broken1arrow** [comment]
> I already pushed 1.18 update to Foundation 6, you can try using that. We'll backport this fix to Foundation 5 in a few days.

you has say we should not use 6 yet.

**Comment by kangarko** [comment]
That's right. Because the courses need updating. Okay, stay on Foundation 5 so it's easier.
